_alerts/2017-07-12-disruptive-forks.md

```diff
@@ -0,0 +1,86 @@
+---
+## This file is licensed under the MIT License (MIT) available on
+## http://opensource.org/licenses/MIT.
+
+title: "Potential disruptive chain forks"
+shorturl: "disruptive-forks"
+active: true
+banner: "Warning: Bitcoin may be unsafe to use on August 1st (click here for details)"
```

Comment:
I think it would be better to say something like:

> Warning: A potential chain split may occur on August 1st.

Also, users don't need to "click here for details" if it's an active link.